web-frontend/src/pages/Home/TransferCoins/SendCoins/SendCoins.tsx

```diff
@@ -1,6 +1,64 @@
-function SendCoins() {
-  // This is a placeholder
-  return <h1>Send coins </h1>;
-}
+import { useState } from 'react';
+import usePost from '../../../../custom/api/usePost';
+import { SendTransactionObject } from '../../../../models/AccountModel';
+import { Button } from '@massalabs/react-ui-kit';
+import { useLocation } from 'react-router-dom';
+import WalletLayout from '../../../../layouts/WalletLayout/WalletLayout';
+export default function SendCoins() {
+  const { state } = useLocation();
+  const nickname = state.nickname;
+  const { mutate, isSuccess } = usePost<SendTransactionObject>(
+    'accounts',
+    nickname,
+    'transfer',
```

Comment:
I am not following this! Why are we passing 3 params to usePost here!? 🤔

Reply:
this is a post query with the url : {{baseURL}}/api/accounts/{{nickname}}/transfer

Reply:
because the function usePost take three params : 
`function usePost<T>(
  resource: string,
  nickname: string,
  path?: string,
): UseMutationResult<T, unknown, T, unknown> {
  // All of our routes using POST method as nickname in the url
  var url = `${import.meta.env.VITE_BASE_API}/${resource}/${nickname}`;

  if (path) {
    url = url.concat(`/${path}`);
  }

  return useMutation<T, unknown, T, unknown>({
    mutationKey: [resource],
    mutationFn: async (payload) => {
      const { data } = await axios.post<T>(url, payload);
      return data;
    },
  });
}

export default usePost;
`

Reply:
I would recommend pinging @peterjah and/or exploring usePost.ts in thyra-plugin-wallet/web-frontend/src/custom/api/usePost.ts

Reply:
> because the function usePost take three params :

That's wrong then! It should take only one: **resource**. If the resource is composed by `something/else/more/etc` usePost does not care. All it needs it the resource complete: "something/else/more".

Reply:
> I would recommend pinging @peterjah and/or exploring usePost.ts in thyra-plugin-wallet/web-frontend/src/custom/api/usePost.ts

No! Here is only FE stuff! usePost is our side. Even in the BE they have these url's

Reply:
yes, here is the curl request:

```
curl --location '127.0.0.1:8080/api/accounts/buildnet/transfer' \
--header 'Content-Type: application/json' \
--header 'Accept: application/json' \
--data '{
    "fee": "10",
    "amount": "10000000",
    "recipientAddress": "AU14nFDPZR5tCkw9Pap6P8Dux2RqAuV6HCee41YVz9ryAgxjH2dX"
}'
```

we needed a optional third argument `path`  for the action /transfer, /sign

Reply:
What **nickname** does there!? What if I want to use **usePost** for another regular request!?

Reply:
I don't know, argument nickname was here before adding argument path https://github.com/massalabs/thyra-plugin-wallet/pull/251/files

Reply:
Ping me if you need help! 

But usePost takes only resource as param. If we need to concatenate something to have the whole resource abc/cde/efg should be before... in the page/component it self.